DockerFile

```diff
@@ -0,0 +1,46 @@
+FROM python:3.8-slim-buster
+
+RUN apt-get update
+RUN apt-get -y install gcc
+
+WORKDIR /rest/BackendApp
+
+COPY ./requirements/base.txt ./
```

Comment:
can we move this to a single copy command.

Reply:
Single Copy command? meaning?

Reply:
requirements folder is already a part of repo....so do we need to copy this sepratly

Reply:
Installing all the dependencies correctly before copying the actual repo.

Reply:
how does it helps?

Reply:
It helps to avoid copying source code before actually checking the pkg dependencies and installing. 
Always preferable to install the dependencies first then bundle the source code at last.

Reply:
does django projects requires any bundling ,, if  am not wrong..
its the python interpretter that directly works on the source code after the app has been bootstrapped...